client/src/views/Room/Room.jsx

```diff
@@ -11,6 +11,7 @@ import MainBox from 'Components/MainBox';
 import Results from 'Components/Results';
 import Timer from 'Components/Timer';
 import { useRoomContext } from 'Contexts/RoomContext';
+import CopyURL from '../../Components/CopyURL/CopyURL';

```

Comment:
ten import można krócej
```suggestion
import CopyURL from 'Components/CopyURL';
```